### PR TITLE
Get polluted location from API

### LINKED ISSUE #17: Get polluted location from API
## Description

Right now, we [fake the polluted location data](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/dev/src/App.tsx#L8). It is time to actually retrieve the data by using [`getAllPollutedLocations`](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/dev/src/backEndClient.ts#L9).

## Steps to take

1. In `src/App.tsx` instead of faking the data, use `getAllPollutedLocations` from `src/backEndClient.ts`:
    1. Refer to [`Example` component](https://github.com/vu-vibedosa/apsitvarkom-fe/blob/dev/src/components/Example.tsx#L11) as an example . You will need `useState` and `useEffect` hooks for this.
    2. Think about what happens if back end is not reachable (display error? show 0 cards? etc.) and what is displayed if the request is slow ([you can simulate this](https://helpdeskgeek.com/networking/simulate-slow-internet-connection-testing/))

## Acceptance criteria

1. Card components rendered in `src/App.tsx` are now using real data coming in from back end API instead of fake objects.

### PR DESCRIPTION
**Pull request by 6ar8nas**
## What was done

- Deleted PollutedLocationType
- Adjusted the type that is returned by getAllPollutedLocations
- Implemented using data from API

The title is currently hardcoded in the PollutedLocationCard. That will need to be adjusted.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,56 +1,34 @@
...
-  ];
+  const [pollutedLocations, setPollutedLocations] = useState<
+    PollutedLocation[]
+  >([]);
+
+  useEffect(() => {
+    getAllPollutedLocations()
+      .then((response) =>
+        setPollutedLocations(
+          response.data.map((loc) => mapToPollutedLocation(loc))
```
Comment (BaVilius) [inline src/App.tsx]: It's a bit of nitpick, but I think we shouldn't shorten variables names for the sake of better readability. Here I would change `loc` to `location`.

File: src/App.tsx [hunk 2]
```diff
@@ -1,56 +1,34 @@
...
+        setPollutedLocations([]);
+      });
+  }, []);
 
   return (
     <Layout>
       <Example />
       <div className="space-y-4 max-w-xl">
-        {fakePollutedLocations.map((pl) => (
+        {pollutedLocations.map((pl) => (
```
Comment (BaVilius) [inline src/App.tsx]: Same here with `pl`.

**Review by BaVilius — CHANGES_REQUESTED** [review]
LGTM, if You don't agree with my naming suggestions, let me know why 😄

**Review by BaVilius — APPROVED** [review]
Great, I think You can merge.

**Review by BenasB — APPROVED** [review]
LGTM, although we'll probably want to add some loading/success/error conditional rendering in the future. Feel free to merge when you can

**Pull request merged**